docs/releases/draft.md

```diff
@@ -0,0 +1,80 @@
+# Release notes
+
+This release contains a refactoring on fluentd. The new architecture moves from fluentd DaemonSet to fluentbit DaemonSet
+with fluentd as StatefulSet.
+
+## Changelog
+
+Changes between `1.3.0` and this release: `X.X.X`
```

Comment:
Just change X.X.X -> 1.4.0 and i think it is ready to be merged!

Am i missing something?